modules/ROOT/pages/salesforce/salesforce-connector-96.adoc

```diff
@@ -0,0 +1,1211 @@
+Query:
+
+* Salesforce Query: Salesforce query to retrieve objects.
+* Parameters: Values for placeholders in the Salesforce query.
+
+Update:
+
+* Type: Salesforce object type.
+* Records: xref:design-center::function-editor-concept.adoc[Function editor expression] to produce a collection of Salesforce objects to be updated.
+
+Delete:
+
+* Records To Delete Ids: xref:design-center::function-editor-concept.adoc[Function editor expression] to produce a collection of Salesforce objects to be deleted.
+
+==== Configure Basic Username Password Authorization
+
+* Username: Enter the Salesforce username.
+* Password: Enter the corresponding password.
+* Security Token: Enter the corresponding security token.
+
+[NOTE]
+Implementing *OAuth 2.0*-based authentication mechanisms involves extra steps, but may be preferred if your service is exposed to external users, as it ensures better security.
+
+==== Configure OAuth 2.0 Authorization
+
+* Display: How to optimize the display: PAGE = Full-page authorization screen (default), POPUP = Compact dialog optimized for modern web browser popup windows, and TOUCH = Mobile-optimized dialog designed for modern smart phones, such as Android and iPhone.
+* Consumer Key: The consumer key for the Salesforce connected app. See https://docs.mulesoft.com/connectors/salesforce/salesforce-connector#create-a-consumer-key[Create a Consumer Key].
+* Consumer Secret: The consumer secret for the connector to access Salesforce.
+* Listener Config: Configuration for the listener, for example, HTTP_Listener_config.
+* Callback Path: Path for the callback, for example, `/callback`
+* Authorize Path: Path for authorization, for example, `/authorize`
+* External Callback URL: Callback URL, for example, `+http://localhost:8085/callback+`
+
+==== Configure OAuth JWT Authorization
+
+* Consumer Key: The consumer key for the Salesforce connected app. See <<Create a Consumer Key>>.
+* Key Store: The path to the key store used to sign data during authentication. Only Java key store format is allowed. See <<Generate a Keystore File>>.
+* Store Password: The password for the keystore.
+* Principal: The Salesforce username that you want to use.
+
+==== Configure OAuth SAML Authorization
+
+* Consumer Key: The consumer key for the Salesforce connected app. See <<Create a Consumer Key>>.
+* Key Store: The path to the key store used to sign data during authentication. Only Java key store format is allowed. See <<Generate a Keystore File>>.
+* Store Password: Key store password
+* Principal: Username of desired Salesforce user
+
+==== Configure OAuth Username Password Authorization
+
+* Consumer Key: The consumer key for the Salesforce connected app. See <<Create a Consumer Key>>.
+* Consumer Secret: The consumer secret for the connector to access Salesforce.
+* Username: Enter the Salesforce username.
+* Password: Enter the corresponding password.
+* Security Token: Enter the corresponding security token.
+
+== Install and Configure in Studio
```

Comment:
If you use the short version, please write Anypoint Studio (Studio) at first occurrence in this file.